examples/combined_ODNP.py

```diff
@@ -309,13 +309,27 @@
             os.remove("temp_ODNP.h5")
             DNP_data.hdf5_write(filename, directory=target_directory)
             final_log.append(
-                "if I got this far, that probably worked -- be sure to move/rename temp_ODNP.h5 to the correct name!!"
+                "if I got this far, that probably worked -- be sure to move/rename temp_ODNP.h5 to the correct name!!")
     logger.info("\n*** FILE SAVED IN TARGET DIRECTORY ***\n")
-    logger.debug(strm("Name of saved data", echo_data.name()))
+    logger.debug(strm("Name of saved data", DNP_data.name()))
     # }}}
     # {{{run IR
+    last_dB_setting = 10
     for j, this_dB in enumerate(T1_powers_dB):
-        p.set_power(this_dB)
+        # {{{ make small steps in power if needed
+        if j == 0:
+            p.set_power(this_dB)
```

Comment:
gen_powerlist returns a list of powers starting at 0 dB. I think it might be useful to have an extra FIR_0dBm to compare with FIR_noPower. Of course this is my opinion and if you think otherwise I can edit to not include FIR_0dBm